docs/modules/ROOT/pages/master-1.adoc

```diff
@@ -0,0 +1,63 @@
+ifdef::context[:parent-context: {context}]
+
+include::{partialsdir}/attributes-1.adoc[]
+
+
+// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+[id='integrating-eventdriven-and-apidriven-applications']
+// If the assembly is reused in other assemblies in a guide, include {context} in the ID: [id='a-collection-of-modules-{context}'].
+
+// Book Title
+= {walkthrough}
+
+//If the assembly covers a task, start the title with a verb in the gerund form, such as Creating or Configuring.
+:context: integrating-eventdriven-and-apidriven-applications
+// The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
+
+ifdef::location[]
+// tag::intro[]
+
+// word count range that fits best is 15-22, with 20 really being the sweet spot. Character count for that space would be 100-125
+// tag::oneliner[]
+Build a simple integration that connects a message queue with an API.
+// end::onliner[]
+
+In a multi-application system, you often find:
+
+* communication between the applications can be difficult due to differing technologies
+* custom solutions can be difficult to create and maintain
+* custom solutions require custom monitoring
+* understanding the relationship between the applications requires deep technical insight to the overall architecture
+
+A more structured approach involves:
+
+* a simple integration that developers can understand intuitively 
+* simple provisioning and configuration
+* built-in monitoring
+
+This walkthrough describes how  to create such an integration, using OpenShift, {messaging-service} and API connectors in Fuse.
+
+++++
+<img src="/images/st0.png" class="img-responsive" alt="integration"> 
+++++
+
+// end::intro[]
+endif::location[]
+
+
+include::{partialsdir}/enmasse.adoc[leveloffset=1]
```

Comment:
For AMQ we don't need any setup in the AMQ console. So I'd guess we'd only need `connections`, `creating-fuse-integration` and `using-integration` here?

I'm also wondering, for `connections`. In Walkthrough One this'll be creating an `AMQ Connection` from Fuse. In Walkthrough One-A it'll be creating an `AMQP Connection` from Fuse.

Their wizards are different. So the steps won't match up between the two walkthroughs. Should be have separate `connection` files for each of the walkthroughs?